server/src/main/scala/controllers/UsersController.scala

```diff
@@ -41,6 +41,22 @@ class UsersController @Inject() (
     } yield Ok(Json.toJson(response))
   }
 
+  def forgotPassword() = handleJsonBody[ForgotPassword.Request] { request =>
+    val body = request.body
+    logger.info(s"Send a link to reset password for user with email: ${body.email}")
+    for {
+      response <- usersService.forgotPassword(body)
+    } yield Ok(Json.toJson(response))
+  }
+
+  def resetPassword() = handleJsonBody[ResetPassword.Request] { request =>
+    val body = request.body
+    logger.info(s"Reset user's password: ${body.token.userId}")
+    for {
+      response <- usersService.resetPassword(body.token.userId, body.password)
```

Comment:
This is wrong, we need to verify the token is correct instead of just trusting it

Reply:
I don't get it. Do you mean we need tokens for this approach? That's what I was planning to do after #114 gets merged